temporal/t.unregister/t.unregister.py

```diff
@@ -77,6 +76,13 @@ def main():
     dbif = tgis.SQLDatabaseInterfaceConnection()
     dbif.connect()
 
+    # modify a stds only if it is in the current mapset
+    # remove all connections to any other mapsets
+    # ugly hack !
+    currcon = {}
+    currcon[mapset] = dbif.connections[mapset]
+    dbif.connections = currcon
```

Comment:
The way to do this in a non-ugly way is to wrap whatever behavior is needed in the library, e.g., by adding function _keep_only_current_mapset_connection_ and then document that `map.is_in_db(dbif, mapset=current_mapset)` can only be used when the above was called. ....or something like that.

Reply:
True, should be easy to add with a simple boolean when initiating a new `SQLDatabaseInterfaceConnection`. But I would leave this for a separate PR.

Reply:
See https://github.com/OSGeo/grass/pull/2448

Reply:
With the PR already open, I'm quite comfortable merging this as is.

Reply:
About `map.is_in_db(dbif, mapset=mapset)`: If you are in mapset A, working with a space-time dataset in mapset B which contains maps from mapset C, the temporal lib must use the TGIS db of the space-time dataset, i.e. of mapset B and ignore the (different or non-existing) TGIS dbs in mapsets A and C. This is the reason for the mapset parameter in `is_in_db()`.

Reply:
Thanks for the explanation.

BTW, the pytest-based tests might be a better tool for testing these inter-mapset things in temporal modules than the gunittest-based tests because you just manage the locations (and mapsets) yourself.